sample/sample.go

```diff
@@ -42,6 +108,19 @@ func LatinHypercube(samples []float64, q dist.Quantiler, src *rand.Rand) {
 	}
 }
 
+// Importancer is a wrapper around the Importance sampling generation method.
+type Importancer struct {
+	Target   dist.LogProber
+	Proposal dist.RandLogProber
+	Src      *rand.Rand
```

Comment:
(no expertise whatsoever. take it with a grain of salt)

is `Importancer.Src *rand.Rand` really needed?
it does not seem to be used by the `Importance(...)` func (and the values are drawn from the `dist.RandLogProber`)